### ISSUE TITLE
[bug]g.Context.FontAtlas.SetDefaultFontSize can't invoke before g.NewMasterWindow

### ISSUE THREAD

**Issue by fyw2020** [AllenDang/giu#629]
### What happend?

g.Context is nil

panic: runtime error: invalid memory address or nil pointer dereference

### Code example

<details><summary><code>main.go</code></summary>

package main

import (
	g "github.com/AllenDang/giu"
)

func loop() {
	g.SingleWindow().Layout(
		g.Style().To(
			g.Label("test"),
		),
	)
}
func main() {
	g.Context.FontAtlas.SetDefaultFontSize(12)
	wnd := g.NewMasterWindow("test", 1024, 768, g.MasterWindowFlagsMaximized)
	wnd.Run(loop)
}

</details>

### To Reproduce

1. Run my demo
2. will see the crash...

### Version

master

### OS

windows

**Comment by fyw2020** [comment]
While the help message say to do so.

// SetDefaultFontSize sets the default font size. Invoke this before MasterWindow.NewMasterWindow(..).

**Comment by gucio321** [comment]
You're right - something is borken inside font atlas right now... `SetDefaultFont` doesn't work anyhow :smile:
